### ISSUE TITLE
Does not install mailcow

### ISSUE THREAD

**Issue by badmarsh** [felixoi/mailcow-hetzner-ansible#1]
I ran the playbook according to README file.

With backups turned off it ends with no errors, creates server in hetzner, ssh connects into it, sets up dns but does not create mailcow directory in /opt and does not start any containers

with backups turned on playbook ends with error Destination directory /opt/mailcow-dockerized does not exist"}

Seems like the part rom mailcow repo clone further is missing

**Comment by felixoi** [comment]
Hey @badmarsh,
thanks for reaching out. I've an idea what's going on maybe.
Did you clone the repo using the `--recursive` flag? This repo contains https://github.com/mailcow/mailcow-ansiblerole as submodule, that's why it's needed. The submodule was initially introduced due to the Ansible Galaxy package being out-of-date. I've fixed that in https://github.com/mailcow/mailcow-ansiblerole/pull/34, so I might looking into removing the submodule in the future. I just added instructions for cloning and updating the repo to the README.

If you cloned the repository correctly, I need some additional information.
Do your logs include the cloning task? If yes, is it being skipped? Example:
```
TASK [mailcow : Clone mailcow git repo] ***********************************************
skipping: [YOUR SERVER IP]
```
